data/_models/shared/Cohorts.csv

```diff
@@ -0,0 +1,17 @@
+tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles
```

Comment:
this is also an example of duplicate. In data catalogue we have 'resources' of which Cohorts is a subclass.

Reply:
Good to know when we are going to merge this